source/usage-examples.txt

```diff
@@ -48,22 +48,49 @@ or you can
 Once you have imported the dataset, you can copy and paste a usage
 example into your development environment of choice. You can follow the
 :doc:`quick start guide </quick-start>` to learn more about getting
-started with the MongoDB Go Driver. Once you've copied a usage example,
-you'll need to edit the connection string to get the example connected to
-your instance of MongoDB:
+started with the MongoDB Go Driver. 
+
+Once you've copied a usage example, you'll need to edit and place
+:ref:`your connection string <usage-examples-connection-string>` in
+:ref:`your environment variable <usage-examples-env-variable>` to get
+the example connected to your instance of MongoDB. 
 
 .. code-block:: go
+   
+   uri := os.Getenv("MONGODB_URI")
+
+.. _usage-examples-env-variable:
+
+Environment Variable
+~~~~~~~~~~~~~~~~~~~~
+
+You can use `GoDotEnv <https://github.com/joho/godotenv>`__ to set up
+your environment variable. 
 
-   // Replace the following with your MongoDB deployment's connection string.
-   uri := "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority";
+When you get to the "Usage" section, add the following application
+configuration in your ``.env`` file at the root of your project: 
+
+.. code-block::
+
+   MONGODB_URI=mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&w=majority
+
+.. note::
+
+   Replace the following with your MongoDB deployment's connection string.
+
+.. _usage-examples-connection-string:
+
+Connection String
+~~~~~~~~~~~~~~~~~
 
 You can use the :guides:`Atlas Connectivity Guide </cloud/connectionstring/>`
 to enable connectivity to your instance of Atlas and find the
-:manual:`connection string </reference/connection-string/>` to replace the
-``uri`` variable in the usage example. If your instance uses
+:manual:`connection string </reference/connection-string/>` to set the
+``MONGODB_URI`` environment variable in the usage example. If your instance uses
 :manual:`SCRAM authentication </core/security-scram/>`, you can replace
 ``<user>`` with your username, ``<password>`` with your password, and
 ``<cluster-url>`` with the IP address or URL of your instance.
 
 For more information about connecting to your MongoDB instance, see the
-Connection Guide.
+<TODO: Connection Guide>.
```

Comment:
Question: Just making sure you knew this TODO was here? Don't want it to go out in prod :)

Reply:
Yes, there's quite a few throughout at the moment -- we'll be doing a pass to update them before putting the docs in prod :)